### ISSUE TITLE
Support for GrabbeAI

### ISSUE THREAD

**Issue by finnbusse** [ollama/ollama#7147]
GrabbeAI is an AI model trained by 10th grade students of the german grammar school Grabbe-Gymnasium Detmold and helps users - especially students and teachers - with their (home)work! From what I know, it is the first LLM made entirely by students!
Since I personally trained the model with some other students with the (financial) assistance from some teachers, you can find the model both on Hugging Face and Ollama Model Hub.
The Hugging Face URL is: https://huggingface.co/grabbe-gymnasium-detmold/grabbe-ai
The Ollama hub URL is: https://ollama.com/grabbe-gymnasium/grabbe-ai

On Hugging Face, the model has about 9 million downloads, on Ollama hub it has more then 300, so I think it could be relevant for a huge group of people! 
We trained the model based on real class test tasks and good homeworks and it works quiet fine. We are actively updating our LLM on Hugging Face and plan to publish a new build until December. This new release will double the trained dataset!

**Comment by rick-github** [comment]
The link to the dataset from the model card is broken - trailing %5D.

**Comment by finnbusse** [comment]
@rick-github It works for me...

**Comment by rick-github** [comment]
![Screenshot 2024-10-09 13 56 47](https://github.com/user-attachments/assets/f19b0f72-a25f-4c07-bb40-82436007bf20)

**Comment by finnbusse** [comment]
@rick-github I don't know why this is there... but this URL is working: https://huggingface.co/datasets/grabbe-gymnasium-detmold/grabbeai
![grafik](https://github.com/user-attachments/assets/3e5b1670-b982-4e3a-b8ee-93f63b7202cb)

**Comment by rick-github** [comment]
This link, not the one on the right hand side.

![Screenshot 2024-10-09 14 03 59](https://github.com/user-attachments/assets/cffe9b41-c442-4ef8-be75-82a4f2d18028)

**Comment by finnbusse** [comment]
Oh this is the same link and the same dataset. But I will try to fix it!

**Comment by rick-github** [comment]
I see you also finetuned qwen2.5 and llama3.1.  Did you find substantial differences between the different models?

**Comment by finnbusse** [comment]
Yes. Those models are experiencing much halluscination, I think this is because of the low amout of parameters the base model was trained with. I am currently working on training the Mistral models. If those give better results, we might change this in the future. But the main differences between qwen2.5 and llama3.1 I noticed was qwen2-5's lower accuracy for other languages than English and Chinese. Llama 3.1 did this quiet well.
